MSGraphSDK/Common/MSGraphSDKVersion.m

```diff
@@ -1,6 +1,6 @@
 // Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
 
-#define MS_GRAPH_SDK_VERSION @"0.10.1"
```

Comment:
Looks like you've decremented the version instead of incrementing? In any case, generally we version just before updating/pushing the cocoapod.